index.js

```diff
@@ -0,0 +1,311 @@
+const newTask = document.getElementById('new-task');
+const addTaskButton = document.getElementById('add-task');
+const editTaskButton = document.getElementById('edit-task');
+const title = document.getElementById('inputTitle');
+const text = document.getElementById('inputText');
+const taskField = document.getElementById('currentTasks');
+const completeField = document.getElementById('completedTasks');
+const priorityFields = document.getElementsByClassName('form-check-input');
+const newSort = document.getElementById('sort-from-new');
+const oldSort = document.getElementById('sort-from-old');
+const save = document.getElementById('save');
+const switcher = document.getElementById('customSwitch1');
+
+let toDo = [];
+let completed = [];
+let index;
+let toDoCount = 0;
+let completedToDoCount = 0;
+
+
+function setTaskTime() {
+    let time = new Date().toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
+    let date = new Date().toLocaleDateString();
+    return time + ' ' + date;
+}
+
+function selectPriority() {
+    for (let i = 0; i < priorityFields.length; i++) {
+        if (priorityFields[i].checked) {
+            return priorityFields[i].value;
+        }
+    }

```

Comment:
use `map` instead

Reply:
For what?

Reply:
For code, don't ask stupid questions please. No need to use `for` cycle in case where we can use map or any other array method